corehq/apps/hqwebapp/static/hqwebapp/js/widgets.js

```diff
@@ -16,6 +17,40 @@ hqDefine("hqwebapp/js/widgets", [
             }, additionalConfig));
         });
 
+        _.each($(".hqwebapp-autocomplete-email"), function (input) {
+            var $input = $(input);
+            $input.select2(_.extend({
+                multiple: true,
+                placeholder: ' ',
+                tags: true,
+                createTag: function (params) {
+                    // Support pasting in comma-separated values
+                    var terms = $.trim(params.term).split(/[, ]\s*/);
```

Comment:
In contrast with the comment above (and PR title), this will allow either space- or comma-delimited values. Was that intended?

I like to add tests in places like this to validate inputs to expected output.

Reply:
That's intentional. The wiki says we support comma-separated values, but my guess is people will interpret that both as `a,b,c` and as `a, b, c` with spaces, and since emails can't have spaces, there's no harm in being looser about the delimiter.

I took a stab at tests and wasn't successful. I can interact programmatically with the select2 by calling `val` on the underlying select element, but I haven't been able to truly mimic how users interact with the text box. Code like `$(".hqwebapp-autocomplete-email").next(".select2-container").find("input").val("abc@dimagi.com").trigger("change")` does insert that value into the text box, but it doesn't trigger the select2 logic that turns the value into a "tag".

Reply:
> people will interpret that both as `a,b,c` and as `a, b, c` with spaces, and since emails can't have spaces, there's no harm in being looser about the delimiter.

Makes sense. But I can't tell from this response if you also intended `a b c` to be interpreted as three separate values (the regex will do that).

> I took a stab at tests and wasn't successful. ... I haven't been able to truly mimic how users interact with the text box.

Yeah, the parsing code is tightly coupled with select2 right now, which makes it hard to test. You could extract the parsing logic to a separate function that has no side effects, and then write tests for that function. Ping me on slack if you want me to elaborate on this more.

Reply:
Yes, interpreting `a b c` as three values is fine.

Added tests for regex in 51c3fd701c2